app/assets/stylesheets/registrations.scss

```diff
@@ -1,7 +1,7 @@
 /*
  * Revised manifest file
  * things from rails-assets
- *= require selectize/selectize.bootstrap3.scss
+ *= require selectize/selectize.bootstrap3
  *= require dataTables/bootstrap/3/jquery.dataTables.bootstrap
```

Comment:
FWIW, I'm 90% sure we don't need dataTables anymore. Not sure if there is a better time to remove it than now 🤷‍♂️